archpkg/PKGBUILD

```diff
@@ -42,7 +28,7 @@ check() {
 
 package_peercoin-cli() {
     pkgdesc="A peer-to-peer network-based digital currency - RPC client"
-    depends=('boost-libs' 'libevent' 'openssl')
```

Comment:
You can't remove openssl, it depends on it.

Reply:
I've removed `openssl` from the dependencies because it is no longer present in the [build-unix.md](https://github.com/peercoin/peercoin/blob/v0.10.0ppc/doc/build-unix.md#dependencies) file in Peercoin 0.10.0. The relevant commits are https://github.com/peercoin/peercoin/commit/d970e378b68776a7fc1f5714303ac8d04e23f181 and https://github.com/peercoin/peercoin/commit/b3a18bce0d180632e570673b4d5e5708d1186e99. I'm not familiar with the Peercoin codebase but when seeing the change I assumed that Peercoin now uses SSL/TLS indirectly via `libevent`. Please note that the Arch Linux `libevent` package depends on `openssl`, so `openssl` is a transitive dependency regardless. If Peercoin indeed still uses `openssl` directly then it should be added back to the documentation.

Reply:
Well, can you please add openssl back to array of dependencies?

Reply:
I think you might have missed my comment [above](https://github.com/peercoin/packaging/pull/9#discussion_r634671419).

Reply:
I've just added it back into documentation, so you do not get confused. Please mind that merge-ins of upstream bitcoin-core code tend to introduce confusion like this all the time. OpenSSL is still required by Peercoin for reward calculation.

Reply:
OK, thanks for clearing this up. I'll update the PR in the following days.